createAccount.php

```diff
@@ -0,0 +1,334 @@
+                $lawncare=0;
+                $photography=0;
+                $gas=0;
+                $grocery=0;
+                $aaaInterest=0;
+                $socialEvents=0;
+                $houseProjects=0;
+            }
+            else{
+                $n=count($services);
+                echo $n . "number of services[] items";
+                for($i=0;$i<$n;$i++){
+                    switch ($services[$i]){
+                        case 'meals':
+                            $meals=1;
+                        case 'lawncare':
+                            $lawncare=1;
+                        case 'housecleaning':
+                            $housecleaning=1;
+                    }
+                }
+            }*/
+
+
+            
+
+            // May want to enforce password requirements at this step
+            $password=$dateOfBirth;
+            //$password = password_hash($args['password'], PASSWORD_BCRYPT);
+
+            if ($errors) {
+                echo '<p>Your form submission contained unexpected input.</p>';
+                die();
+            }
+
+            $optional=array('allergies','sibling_info','can_share_contact_info',
+            'family_info','how_did_you_hear','address2');
+
+            if($args['allergies']){
+                $allergies=$args['allergies'];
+            }
+            else{
+                $allergies="";
+            }
+
+            if($args['sibling_info']){
+                $sibling_info=$args['sibling_info'];
+            }
+            else{
+                $sibling_info="";
+            }
+
+
+            if($args['can_share_contact_info']){
+                $can_share_contact_info=$args['can_share_contact_info'];
+            }
+            else{
+                $can_share_contact_info="";
+            }
+
+            if($args['family_info']){
+                $family_info=$args['family_info'];
+            }
+            else{
+                $family_info="";
+            }
+            
+            if($args['how_did_you_hear']){
+                $how_did_you_hear=$args['how_did_you_hear'];
+            }else{
+                $how_did_you_hear="";
+            }
+            
+            if($args['address2']){
+                $address=$address.", ".$args['address2'];
+            }
+
+            
+            if(isset($_POST['meals'])){
+                $meals=1;
+            }
+            else{
+                $meals=0;
+            }
+            if(isset($_POST['gascards'])){
+                $gascards=1;
+            }
+            else{
+                $gascards=0;
+            }
+            if(isset($_POST['lawncare'])){
+                $lawncare=1;
+            }
+            else{
+                $lawncare=0;
+            }
+            
+            if(isset($_POST['housecleaning'])){
+                $housecleaning=1;
+            }
+            else{
+                $housecleaning=0;
+            }
+            /*
```

Comment:
Is this to line 271 needed it's commented out.

Reply:
Lines 251 to 271 were again part of the testing of the checkbox parsing. Lines 271 to 285 were to ensure the code ran without errors. Lines 251 to 285 can all be deleted.